packages/coqide/coqide.8.12.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1-only"
+synopsis: "IDE of the Coq formal proof management system"
+description: """
+CoqIDE is a graphical user interface for interactive development
+of mathematical definitions, executable algorithms, and proofs of theorems
+using the Coq proof assistant.
+"""
+
+depends: [
+  "coq" {= version}
+  "lablgtk3-sourceview3" {>= "3.1.0"}
```

Comment:
@RalfJung @ejgallego : Is there a good reason for the restriction to lablgtk 3.1.0? For a reason yet to be investigated this version does not build on windows, but 3.0.beta5 does and it seems to work fine.

Reply:
That's from https://github.com/ocaml/opam-repository/pull/16876#discussion_r460383730; there was a specific reason we banned beta7, but beta8 was fine again, and I suspect it was fixed on the Coq side as well.

Reply:
Sorry: that's to say "no, there's no strong reason for the restriction to be that strict".

Reply:
In fact, I should have searched the original issue. Indeed, beta7 needn't be blacklisted since 
https://github.com/coq/coq/pull/11234, which was merged in 8.11.0; only 8.10.* need the exclusion, as I noted in https://github.com/garrigue/lablgtk/pull/100#issuecomment-560587640.

Reply:
Summing up: no, beta7 had to be banned only for coqide 8.10.*. No comment on potential other problems.
Coq 8.11.0 should be compatible with beta7 since https://github.com/coq/coq/pull/11234.

=== Below the former comments which I dropped.

1. That's from https://github.com/ocaml/opam-repository/pull/16876#discussion_r460383730; there was a specific reason we banned beta7, but beta8 was fine again, and I suspect it was fixed on the Coq side as well.

2. Sorry: that's to say "no, there's no strong reason for the restriction to be that strict".

3. In fact, I should have searched the original issue. Indeed, beta7 needn't be blacklisted since 
https://github.com/coq/coq/pull/11234, which was merged in 8.11.0; only 8.10.* need the exclusion, as I noted in https://github.com/garrigue/lablgtk/pull/100#issuecomment-560587640.

Reply:
Okay, I opened https://github.com/ocaml/opam-repository/pull/16881 to relax the version bound.